src/checkUrl.js

```diff
@@ -0,0 +1,36 @@
+const cases = [
+  'SNAKE',
+  'KEBAB',
+  'CAMEL',
+  'PASCAL',
+  'UPPER',
+];
+
+const checkUrl = (text, toCase) => {
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. '
+      + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
It's good practice to use `template literals` like this
```suggestion
      message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
```
It doesn't require to escape special characters (text inside will be as it is including spacing). Also you can include variables inside.